app/build.gradle

```diff
@@ -123,4 +123,5 @@ dependencies {
 
     implementation 'com.google.code.gson:gson:2.10.1'
 
+    implementation("androidx.core:core-splashscreen:1.0.1")
```

Comment:
Why this is required?

Reply:
sorry i miss the comment for splash screen

Reply:
I mean why you added that dependency? And where you used that in code ?

Reply:
i use this code in the theme

Reply:
<item name="windowSplashScreenAnimatedIcon">@drawable/nextgen_logo</item>
 <item name="windowSplashScreenBackground">@color/black</item>
 <item name="postSplashScreenTheme">@style/Theme.NextGen</item>


these are the code

Reply:
yes , error during the build is failed linking refrences

![image](https://github.com/user-attachments/assets/9a3e0459-b6c2-409e-940c-ba4d39c0f1a3)


![image](https://github.com/user-attachments/assets/7d0dafc7-d385-4262-9ee5-f00aa0278f28)


these are the errors

Reply:
Got it, thanks